crates/language/src/target_language.rs

```diff
@@ -227,7 +232,11 @@ impl PatternLanguage {
             "sql" => Some(Self::Sql),
             "vue" => Some(Self::Vue),
             "toml" => Some(Self::Toml),
-            "php" => Some(Self::Php),
+            "php" => match flavor {
+                Some("html") => Some(Self::Php),
+                Some("only") => Some(Self::PhpOnly),
+                _ => Some(Self::PhpOnly),
```

Comment:
I think we might want HTML as the default actually, since it seems like the most common thing.

Reply:
ok, I'll change the default to html